EST.MIT.InvoiceImporter.Function/Functions/ImporterFunctions.cs

```diff
@@ -57,4 +71,21 @@ public ImporterFunctions(IAzureBlobService azureBlobService, IAzureTableService
             throw;
         }
     }
+
+    private Notification CreateNotificationRequest(ImportRequest importMessage)
```

Comment:
Good idea. this might needs to be the ImportRequestEntity, and fed called AFTER the ImportRequest is saved to table storage as an ImportRequestEntity. This is because the Web doesn't set the ID in the ImportRequest - the Importer does. It's only in ImportRequest so it can be passed back to Web when we are displaying the list of uploads.

Reply:
We discussed this, and I've tested it out. Works OK.